### DISCUSSION TITLE
MIssing Perm

### DISCUSSION THREAD

**Discussion by SpaceLeft** [SudhanPlayz/Discord-MusicBot#498] [Ideas]
Add Missing Perm For commands embed

**Comment by ItzSudhan** [comment]
what do you mean?

**Comment by supratikmahato** [comment]
> what do you mean?

I was also going to say that

**Comment by aryan-tah** [comment]
> what do you mean?

I think he means to say to add a permission handler where you check the bot's permissions in every guild for each command, like if one guild removes MANAGE_CHANNELS perm, the bot should send `Missing Permission - MANAGE_CHANNELS` if any of its command require that perm
